api/src/methods/get_module.rs

```diff
@@ -0,0 +1,234 @@
+use {
+    crate::{json_utils::parse_params_3, jsonrpc::JsonRpcError},
+    umi_app::{ApplicationReader, Dependencies},
+    umi_shared::primitives::{Address, ToMoveAddress},
+};
+
+pub async fn execute<'reader>(
```

Comment:
Suggestion: We should add comments with example request params, since this is a non-standard RPC.